### PR TITLE
[WIP] Update hubtools to init file

### PR DESCRIPTION
**Pull request by shiyu22**
Signed-off-by: shiyu22 <shiyu.chen@zilliz.com>

### CODE REVIEW

**Comment by junjiejiangjjj** [comment]
We could add a towhee/bin directory, and move `main` to  bin/hub.py.

File: towhee/hub/hub_tools.py [hunk 1]
```diff
@@ -552,7 +581,8 @@ def generate_repo_yaml(author: str, repo: str) -> None:
         yaml.dump(data, outfile, default_flow_style=False, sort_keys=False)
 
 
-def main(argv):
+def main():
```
Comment (junjiejiangjjj) [inline towhee/hub/hub_tools.py:584]: The `main` function is so large,  we could use `argparse` lib to optimize it.

File: towhee/hub/hub_tools.py [hunk 2]
```diff
@@ -423,47 +425,74 @@ def create_repo(repo: str, token: str, repo_type: str) -> None:
...
+        op_choice = input('Is it NNOperator (there are neural network or model related codes)? '
+                          '[Y|n, press Enter will default to \'No\']  ')
+        if op_choice.lower() in ['yes', 'y']:
+            nn_framework = input('What\'s the framework of your model? [press Enter will default to \'pytorch\']  ')
+            repo_temp = 'nnoperator-template'
+
+    print('Clone the repo and initialize it with template...')
+    url = f'https://towhee.io/{author}/{repo}.git'
+    url_temp = f'https://towhee.io/towhee/{repo_temp}.git'
+    git.Repo.clone_from(url=url, to_path=repo, branch=tag)
```
Comment (filip-halt) [inline towhee/hub/hub_tools.py:482]: im thinking that this functionality should be added in a seperate tool to avoid requiring git for all towhee users.

**Comment by fzliu** [comment]
Can you squash this into one commit?

File: towhee/hub/hub_tools.py (hunk 1 above)
Comment (shiyu22) [reply towhee/hub/hub_tools.py:584]: working in progress #440 .

File: towhee/hub/hub_tools.py (hunk 2 above)
Comment (shiyu22) [reply towhee/hub/hub_tools.py:482]: Actually, the hub tool is developed for contributors, so they have to install git and git-lfs, of course the README will state that.

**Comment by shiyu22** [comment]
Submit another PR #449

**Pull request closed without merging**